public/js/src/client.js

```diff
@@ -1,10 +1,21 @@
+/*
+	You're doing everything in the ready state.  
+	Is there anything that could be done asynchronously,
+	while the document is loading?  The initial $.get, 
+	for instance?  Could this retrieve data, then wait 
+	till DOM ready to insert it?
+*/
 $(document).ready(function(){
+	"use strict";
 
 	var OPERATIONS = {};
 
+	var $window = $(window);
```

Comment:
If you're ever going to use a jQuery object twice, cache it like so.  The $ at the head of the var signifies the cached jQuery-ness of the object.

When working with the developers at my job (and when feeling a bit cruel), I have them put a breakpoint at the beginning of the jQuery object's creation, and step through each step in the jQuery to see how many processes that actually takes.  It's one of the main reasons I don't fancy jQuery - it's very easy to be non-performant without thinking about it.